### PR TITLE
[WIP] prepare GSoc infrastructure for multiple years; archive 2017

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -1,81 +1,17 @@
...
-
-
-[Full list of Proposal Ideas](/gsoc/summary.html)
-
-[Full list of Mentors](/gsoc/mentors.html)
-
-## Contact information (for students)
-Please do not hesitate to contact us if you are planning to apply for any of the above projects:
-HSF GSoC mailing list:[hep-software-foundation-google-summer-of-code@googlegroups.com](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com)  no subscription needed
+The information of last year can be found [here](#Projects) and [here](/gsoc/2017/index.html). For 2018 the HSf is again applying for participation in the program.
```
Comment (etejedor) [inline _activities/gsoc.md]: HSf --> HSF

Comment (graeme-a-stewart) [reply _activities/gsoc.md]: Also, the first link is now dead and should be removed.

File: _activities/gsoc.md [hunk 2]
```diff
@@ -1,81 +1,17 @@
...
+# ![CERN](/images/CERN-logo.jpg){:height="100px"} ![HSF](/images/hsf_logo_angled.png){:height="100px"} Google Summer of Code
 
 ## Introduction
 
-[Google Summer of Code 2017](https://developers.google.com/open-source/gsoc/) is a program that allows students to contribute to development of open-source projects, mentored by participating organizations.
+[Google Summer of Code](https://developers.google.com/open-source/gsoc/) is a program that allows students to contribute to development of open-source projects, mentored by participating organizations.
 
 Particle physics is an exciting field where large collaborations of scientists collect and analyze the data from high-energy physics experiments, such as the Large Hadron Collider, hosted by the CERN laboratory in Geneva, Switzerland. The questions that we collectively ask are: what are the fundamental blocks that make up our Universe? What is the nature of dark matter and dark energy?  What is the nature of the asymmetry between matter and antimatter? What was early Universe like? To answer these questions, particle physicists build software to simulate and analyze what happens in particle physics detectors.
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md]: It would probably read better as a bulleted list, instead of a large text block

**Comment by jouvin** [comment]
@graeme-a-stewart I have not checked what you did but according to the number of changed files, I suspect you created additional directories... I haven't thought at it in details but may be adding a new collection for this year, letting the previous one untouched would have been easier... I'll try to look at what you did in more details... I had also a few issues open to improve the proposal/project handling and remove some of the remaining duplication: that would have been pretty easy but I had no time to look at it!

**Comment by graeme-a-stewart** [comment]
Hi @jouvin - @hegner is the original author :-)

What he did was to move all the old proposals into a 2017 directory, which I think makes a lot more sense than leaving them in the upper level "gsoc" directory. So although there are a lot of changes, most are trivial ones.

**Comment by graeme-a-stewart** [comment]
Closing this in favour of https://github.com/HEP-SF/hep-sf.github.io/pull/206, which contains more fixes.

**Pull request closed without merging**